udata/core/spatial/models.py

```diff
@@ -29,6 +29,11 @@ class GeoLevel(db.Document):
     parents = db.ListField(db.ReferenceField('self'))
 
 
+class GeoZoneQuerySet(db.BaseQuerySet):
+    def valid(self):
+        return self(validity__end='')
```

Comment:
Instead of empty string, it might be better if absent or set to `null` and so `return self(validity__end=False)` (I think)